protocol/shannon/mode_centralized.go

```diff
@@ -105,17 +101,15 @@ func (p *Protocol) getCentralizedGatewayModeSessions(
 	)
 	logger.Debug().Msg("fetching the list of owned apps.")
 
+	ownedAppsForService, ok := p.ownedApps[serviceID]
```

Comment:
Please add a TODO here: if an app is changed, e.g. staked for a different service, we will need a PATH restart.

Reply:
I've updated with a TODO. 

To be clear, do you mean this TODO should cover implementing a mechanism to avoid needing to restart PATH in the case of restaking an app or do we just want it to be a notice that it is required